### ISSUE TITLE
Report option for spam messages

### ISSUE THREAD

**Issue by eneerhut** [openstreetmap/openstreetmap-website#3127]
Received this message in my OSM inbox. It'd be great to have a report button that could link with whatever process has been set up to report users in other scenarios.

![image](https://user-images.githubusercontent.com/11349863/110968277-2cd5fe00-830c-11eb-9d20-7fe51d1b8e46.png)

**Comment by eneerhut** [comment]
Yes, I see that is an option, but edited comment because I think there is value in having it directly on the message. Probably a nice to have.

**Comment by tomhughes** [comment]
Reporting a message doesn't (on the admin side) actually add anything at the moment because there is no way for an admin to see a reported message (or any other user's messages) from the web site.

That's possibly something we should fix (having to go look at the database is a pain) but it's a rather bigger job.

**Comment by jidanni** [comment]
> Just click on the user name to go to their profile and there is a link to report them - several people did and the account has been closed.

Even having to click on the message to open it to look around for a (non existent) 'report spam' button violates the "never open spam messages" principle of Computer Science.

**Comment by jidanni** [comment]
https://www.openstreetmap.org/messages/945567 is another spam message.
